README.md

```diff
@@ -80,14 +80,24 @@ Link: [https://d3js.org](https://d3js.org)
 5. [Recommended] Install virtualenv.
 6. [Recommended] If you are using IntelliJ, install the [PyLint Plugin](https://plugins.jetbrains.com/plugin/11084-pylint/).
 
-#### Step 2: Create the database
+#### Step 2: Initialize the database
 1. Start PostgreSQL.
 
 2. Create a database called research:
 ```
 createdb research
 ```
 
+3. Navigate to the backend directory: 
+```
+cd backend
+```
+
+4. Initialize database migrations:
+```
+python app.py db init
```

Comment:
I am actually not 100% sure that we need this step. Because when deploying to Heroku I "messed it up" by _just_ running `db upgrade` and it seems to work. But I am sure that `upgrade` is needed (applies all pending migration scripts). Need to check this `init` thing in the docs. Give me 5min

Reply:
hmm, I would say we don't necessarily need `init` every time (https://flask-migrate.readthedocs.io/en/latest/). This is only for creating the `migrations` folder and `upgrade` is the thing to actually set up the database. At least in my humble understanding :D

Reply:
That's true... so would you just delete it, i.e. close this task? 
Is there anything else you would like to see in the Readme regarding migrations?

Reply:
oh no no, don't delete it :D 
I meant to exchange `init` with `upgrade` :sweat_smile:

Reply:
We definitely need this in the README